packages/mask/popups/modals/GasSettingModal/GasSettingDialog.tsx

```diff
@@ -132,6 +138,11 @@ export const GasSettingDialog = memo<GasSettingDialogProps>(function GasSettingM
         return
     }, [gasPrice, gasOptions, isSupport1559])
 
+    const gasLimitError = useMemo(() => {
+        if (isGreaterThanOrEqualTo(gasLimit, MinGasLimit) && isLessThanOrEqualTo(gasLimit, MaxGasLimit)) return
```

Comment:
Why set a max limit🤔? Is there any reference for this value?

Reply:
https://etherscan.io/blocks

A block can only use 30,000,000 gas. tx exceed this limit won't be packed into the block

Reply:
> Each block has a target size of 15 million gas

https://ethereum.org/developers/docs/gas

so 15,000,000 gas is a reasonable max value, which only allows 1 tx in 1 (minimal) block.

Reply:
> OK, but it's not necessarily the same case with other chains. eg: [bscscan.com/blocks](https://bscscan.com/blocks).

hmmm ok, I'll remove that limit